src/app/apiTest/hooks/todoQueries.ts

```diff
@@ -1,9 +1,48 @@
-import { useQuery, UseQueryResult } from '@tanstack/react-query';
-import { getAllByUserId, getOneByTodoId } from '@/app/apiTest/calls/todoCalls';
+import { useMutation, UseMutationResult, useQuery, UseQueryResult } from '@tanstack/react-query';
+import {
+  createTodo,
+  deleteTodo,
+  getAllByUserId,
+  getOneByTodoId,
+  updateTodo,
+  updateTodoTime,
+} from '@/app/apiTest/calls/todoCalls';
 import { SelectTodo } from '@/db/schema/todos';
 
+export const useCreateTodo = (): UseMutationResult<
+  SelectTodo,
+  Error,
+  { userId: string; title: string; content: string; startTime: string; endTime: string; color: string }
+> =>
+  useMutation({
+    mutationFn: ({ userId, title, content, startTime, endTime, color }) =>
+      createTodo(userId.toString(), title, content, startTime, endTime, color),
+  });
 export const useGetAllTodos = (userId: string): UseQueryResult<SelectTodo[], Error> =>
   useQuery({ queryKey: ['todos', userId], queryFn: () => getAllByUserId(userId), enabled: !!userId });
 
 export const useGetOneTodo = (todoId: string): UseQueryResult<SelectTodo, Error> =>
   useQuery({ queryKey: ['todo', todoId], queryFn: () => getOneByTodoId(todoId), enabled: !!todoId });
+
+export const useUpdateTodo = (): UseMutationResult<
+  SelectTodo,
+  Error,
+  { todoId: string; title: string; content: string; startTime: string; endTime: string; color: string }
+> =>
+  useMutation({
+    mutationFn: ({ todoId, title, content, startTime, endTime, color }) =>
+      updateTodo(todoId, title, content, startTime, endTime, color),
+  });
+export const useUpdateTodoTime = (): UseMutationResult<
```

Comment:
```suggestion
  });
  
export const useUpdateTodoTime = (): UseMutationResult<
```
다른 사람이 볼 때, 이것을 읽을 때도 고려해주면 좋을 것 같아요 ~
예시로 함수별로 줄바꿈을 넣는 등이 있을 것 같아요!

Reply:
네 적용했습니다:)